src/NHSD.BuyingCatalogue.Database/PostDeployment/IntegrationTests/ClearData.sql

```diff
@@ -3,8 +3,8 @@
 
     TRUNCATE TABLE dbo.FrameworkCapabilities;
     
-    DELETE FROM dbo.CataloguePriceTier;
-    DELETE FROM dbo.CataloguePrice;
+    --DELETE FROM dbo.CataloguePriceTier;
+    --DELETE FROM dbo.CataloguePrice;
```

Comment:
Can you delete these rather than comment them out?

Reply:
Sorry, thought I did, but it must not have saved.